src/public/secure/admin.ejs

```diff
@@ -301,6 +317,28 @@
                                 reloadTable(["entries", "ips"]);
                             }
                         },
+                        {
+                            extend: 'selectedSingle',
+                            text: 'Details',
+                            attr: {id: 'detailsButton'},
+                            action: async (e, dt) => {
+                                const selectedRow = dt.row({selected: true}).data();
+                                const container = document.getElementById('fileDetailsModelInfoContent');
+                                container.innerHTML = '';
+                                addDetailInfo("ID", selectedRow.id);
+                                addDetailInfo("File Name", selectedRow.fileName);
+                                addDetailInfo("File Ext", selectedRow.fileExtension);
+                                addDetailInfo("Original Name", selectedRow.originalFileName);
+                                addDetailInfo("Media Type", selectedRow.mediaType);
+                                addDetailInfo("File Size", selectedRow.fileSize);
+                                addDetailInfo("Created At", customDate()(selectedRow.createdAt, 'display', selectedRow));
+                                addDetailInfo("Expires", selectedRow.expires);
+                                addDetailInfo("IP", selectedRow.ip);
+                                addDetailInfo("URL", selectedRow.url);
+                                const detailsModal = new bootstrap.Modal(document.getElementById('fileDetailsModel'), {keyboard: false});
+                                detailsModal.show();
```

Comment:
move this out. i don't think this needs to happen for each time the button is clicked, move it to the `loadDt` function

Reply:
Implemented